### PR TITLE
chore: Add support for Scala 2.13.10

### PR DESCRIPTION
**Pull request by tgodzik**
Waiting for ScalaJS and Native releases

Fixes https://github.com/scalameta/scalameta/issues/2902

### CODE REVIEW

**Review by tanishiking — COMMENTED** [review]
I left one question, but otherwise looks good to me:)

File: semanticdb/scalac/library/src/main/scala-2.13.10/scala/meta/internal/semanticdb/scalac/SemanticdbReporter.scala [hunk 1]
```diff
@@ -0,0 +1,27 @@
+package scala.meta.internal.semanticdb.scalac
+
+import scala.reflect.internal.util.Position
+import scala.tools.nsc.reporters.{FilteringReporter, StoreReporter}
+import scala.tools.nsc.Settings
+
+class SemanticdbReporter(underlying: FilteringReporter)
```
[the remark this answers is not in the document]
Comment (tgodzik) [reply semanticdb/scalac/library/src/main/scala-2.13.10/scala/meta/internal/semanticdb/scalac/SemanticdbReporter.scala:7]: och, I don't really remember, we added this a long while ago :fearful:

[the remark this answers is not in the document]
Comment (tgodzik) [reply semanticdb/scalac/library/src/main/scala-2.13.10/scala/meta/internal/semanticdb/scalac/SemanticdbReporter.scala]: Actually, I can remove 2.13.0 and the separate SemanticdbReporter

[the remark this answers is not in the document]
Comment (tanishiking) [reply semanticdb/scalac/library/src/main/scala-2.13.10/scala/meta/internal/semanticdb/scalac/SemanticdbReporter.scala]: Sorry, I realized this is `scalameta/scalameta`, not `scalameta/metals` 🤣  but I'm happy if you realized that we can drop `SemanticdbReporter` by dropping `2.13.0` by my comment... (which is acceptable :+1: as we don't ship semanticdb plugin for `2.13.0` anyway).

**Review by tanishiking — APPROVED** [review]
LGTM!

**Comment by kitbellew** [comment]
@tgodzik could this be related to the missing Diagnostics from #2993? I ran `git diff v4.6.0 semanticdb/`, and there are only two more commits affecting that source tree (#2942 and #2943), and they _look_ unlikely.

**Comment by tgodzik** [comment]
I thought so, but I used the same commit to backpublish for Scalameta 4.6.0, so the result should be the same. Also I though we had tests to prevent that, but they might not test every version properly.

**Pull request merged**